index.bs

```diff
@@ -2523,26 +2447,12 @@ partial interface MLGraphBuilder {
     element-wise unary operation of the input tensor. The shape of the output
     tensor is the same as the shape of input tensor.
 </div>
-<div>
-    **Operation types:**
-        - *abs*: Compute the absolute value of the input tensor, element-wise.
-        - *ceil*: Compute the ceiling of the input tensor, element-wise.
-        - *cos*: Compute the cosine of the input tensor, element-wise.
-        - *exp*: Compute the exponential of the input tensor, element-wise.
-        - *floor*: Compute the floor of the input tensor, element-wise.
-        - *log*: Compute the natural logarithm of the input tensor, element-wise.
-        - *neg*: Compute the numerical negative value of the input tensor, element-wise.
-        - *sin*: Compute the sine of the input tensor, element-wise.
-        - *tan*: Compute the tangent of the input tensor, element-wise.
-</div>
-
 <details open algorithm>
-
   <summary>
     To <dfn for="MLGraphBuilder" data-lt="element-wise-unary-op">create element-wise unary operation</dfn> given |op| and |input|, run the following steps:
   </summary>
   <div class=algorithm-steps>
-    1. [=Assert=]: |op| is one of "abs", "ceil", "cos", "exp", "floor", "log", "neg", "sin", "tan".
+    1. [=Assert=]: |op| is one of "abs", "ceil", "copy", "cos", "exp", "floor", "log", "neg", "reciprocal", "sin", "sqrt", "tan".
```

Comment:
Missed "erf" op.

Reply:
Fixed.

Reply:
Fixed in https://github.com/webmachinelearning/webnn/pull/478/commits/873eec1e656253ed5e5f600f0ebf7e5fd12b9746